src/client/src/pages/DataView360/View/components/Adoptions.js

```diff
@@ -89,27 +90,34 @@ class Adoptions extends Component {
                     headerText={headerText + headerAddition}
                     emojiIcon={<PetsIcon color='primary' fontSize='inherit' />}
                 >
-                    <Grid item>
-                        <IconButton style={{ 'padding': 0, 'paddingLeft': 5 }} color="primary" aria-label="link" href={shelterLuvPersonURL} target="_blank">
-                            <LinkIcon />
-                        </IconButton>
-                    </Grid>
+                    {shelterluvShortId &&
+                        <Grid item>
+                            <IconButton style={{ 'padding': 0, 'paddingLeft': 5 }} color="primary" aria-label="link" href={shelterLuvPersonURL} target="_blank">
+                                <LinkIcon />
+                            </IconButton>
+                        </Grid>
+                    }
```

Comment:
This is a link to shelterluv that just gives a 404 if the ID doesn't exist. I figured we didn't want to send people there.